x-pack/plugins/cases/public/components/case_action_bar/index.tsx

```diff
@@ -69,6 +68,20 @@ const CaseActionBarComponent: React.FC<CaseActionBarProps> = ({
     [onUpdateField]
   );
 
+  const { data: userActionsData, isLoading: isLoadingUserActions } = useGetCaseUserActions(
```

Comment:
moving this from the case view directlly here. With react-query we don't need to worry on querying the same twice on the same render.

Reply:
Those kind of comments  are too specific to the implementation and how the library works and I find it more a matter of getting used to the library than having to comment the code.

Imagine we have to comment every react hook we use.